src/wmnds/assets/sass/_utilities.scss

```diff
@@ -100,43 +100,43 @@ body {
   .wmnds {
     // Generate all margin classes
     &-m {
-      @include x-mar-pad('');
+      @include x-mar-pad("");
 
       &-t {
-        @include x-mar-pad('-top');
+        @include x-mar-pad("-top");
       }
 
       &-r {
-        @include x-mar-pad('-right');
+        @include x-mar-pad("-right");
       }
 
       &-b {
-        @include x-mar-pad('-bottom');
+        @include x-mar-pad("-bottom");
       }
 
       &-l {
-        @include x-mar-pad('-left');
+        @include x-mar-pad("-left");
       }
     }
 
     // Generate all padding classes
     &-p {
-      @include x-mar-pad('', 'padding');
+      @include x-mar-pad("", "padding");
 
       &-t {
-        @include x-mar-pad('-top', 'padding');
+        @include x-mar-pad("-top", "padding");
       }
 
       &-r {
-        @include x-mar-pad('-right', 'padding');
+        @include x-mar-pad("-right", "padding");
       }
 
       &-b {
-        @include x-mar-pad('-bottom', 'padding');
+        @include x-mar-pad("-bottom", "padding");
       }
 
       &-l {
-        @include x-mar-pad('-left', 'padding');
+        @include x-mar-pad("-left", "padding");
       }
     }
   }
```

Comment:
Can we ignore this file changes?
the preettier should be using the single quotes instead double quotes (on this project)..

Reply:
@catiarodriguescosta This file is fine as the .sass-lint enforces the rules for sass/css and in this case it enforces double quotes.

Prettier, along with Eslint enforces rules for Javascript.